resources/dicts/events/new_cat/general.json

```diff
@@ -4,11 +4,12 @@
         "biome": ["any"],
         "camp": ["any"],
         "season": ["any"],
-        "tags": ["adoption"],
+        "sub_types": ["adoption"],
```

Comment:
make sure it's `sub_type` not `sub_types`

Reply:
Fixed in b7da8fe, thank you for notifying me